README.md

```diff
@@ -138,7 +138,7 @@ Branch filters allow filtering commit notifications. By default when you subscri
 - `/github subscribe org/repo commits`  for commit notifications from a default branch.
 - `/github subscribe org/repo commits:*`  for commit notifications across all the branches.
 - `/github subscribe org/repo commits:myBranch`  for commit notifications from a specific branch.
-- `/github subscribe org/repo commits:users/*`  for commit notifications from a pattern of branches.
+- `/github subscribe org/repo commits:users/*`  for commit notifications from a pattern of users.
```

Comment:
I don't think this change is correct. I think it's just a silly example where branches are named `users/someone/something`

Reply:
I see! I'll close this comment up then. :)